content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Security policy examples
+description: Shows common examples of using Istio security policy.
+weight: 60
+owner: istio/wg-security-maintainers
+test: n/a
```

Comment:
I'm thinking this is more of a `no` than `n/a`. This page could have tests of the various snippets.

Reply:
updated to `no`.